Trinap/Sources/Presenter/Auth/CreateUser/CreateUserViewController.swift

```diff
@@ -11,44 +11,126 @@ import UIKit
 import RxCocoa
 import RxSwift
 import SnapKit
+import Than
 
 final class CreateUserViewController: BaseViewController {
     
-//    // MARK: - UI
-//
-//    // MARK: - Properties
-//    private let viewModel: CreateUserViewModel
-//
-//    // MARK: - Initializers
-//    init(viewModel: CreateUserViewModel) {
-//        self.viewModel = viewModel
-//
-//        super.init()
-//    }
+    // MARK: - UI
+    private lazy var titleLabel = UILabel().than {
+        $0.text = "만나서 반가워요!\n트리냅과 추억을 만들어 보세요"
+        $0.numberOfLines = 0
+        $0.font = TrinapFontFamily.Pretendard.bold.font(size: 24)
+    }
+    
+    private lazy var subTitleLabel = UILabel().than {
+        $0.text = "트리냅 이용을 위해 닉네임을 설정해 주세요."
+        $0.font = TrinapFontFamily.Pretendard.regular.font(size: 14)
+    }
+    
+    private lazy var nicknameTextFieldView = NicknameTextFieldView()
+    
+    private lazy var signUpButton = TrinapButton(
+        style: .primary
+    ).than {
+        $0.setTitle("가입 완료", for: .normal)
+        $0.setTitleColor(TrinapAsset.white.color, for: .normal)
+        $0.titleLabel?.font = TrinapFontFamily.Pretendard.bold.font(size: 16)
+    }
+
+    // MARK: - Properties
+    private let viewModel: CreateUserViewModel
+
+    // MARK: - Initializers
+    init(viewModel: CreateUserViewModel) {
+        self.viewModel = viewModel
+
+        super.init()
+    }
     
     // MARK: - Methods
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .blue
     }
     
-//    override func configureHierarchy() {
-//        [
-//            <#view#>,
-//        ].forEach { self.view.addSubview($0) }
-//    }
-//
-//    override func configureConstraints() {
-//        <#view#>.snp.makeConstrains { make in
-//            make.<#code#>
-//        }
-//    }
-    
-//    override func configureAttributes() {
-//        <#code#>
-//    }
-//
-//    override func bind() {
-//        <#code#>
-//    }
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+    
+    override func configureHierarchy() {
+        [
+            titleLabel,
+            subTitleLabel,
+            nicknameTextFieldView,
+            signUpButton
+        ].forEach { self.view.addSubview($0) }
```

Comment:
앗 제가 말씀을 안드렸는데 `UIView`에 `addSubviews` 메서드 추가해 놓았습니다!
나중에 뷰 짜실 때 사용하시면 좋을 것 같습니다!

```suggestion
        self.view
            .addSubviews([titleLabel, subTitleLabel, nicknameTextFieldView, signUpButton])
```